### ISSUE TITLE
It seems that the pretrain model you provide has gap on modelnet40 

### ISSUE THREAD

**Issue by Zscozer** [MohamedAfham/CrossPoint#14]
Hi, I used your pretrain model directly test linear accuracy on modelnet40, it got 90.27%, same as I runed train_crosspoint.py without any initialize, But the result you mentioned in your paper can get 91.2%. So I want to know are there any tricks in your codes. Or It means I should train based on your pretrain model?  I look forward to your answers

**Comment by SL-zZZ** [comment]
I even got the lower accuracy than yours, 89.7%. And I run the original file with the original parameters.

**Comment by MohamedAfham** [comment]
Hi @Zscozer and @UnderTheMangoTree ,

I think the results will vary based on the random seed as well. Sometimes, you may get results more than the one reported in the paper. We reported the average accuracy over multiple runs and to be in fair comparison with other works we didn't report the standard deviation.

**Comment by SL-zZZ** [comment]
Thanks for your reply, the performance of model indeed exits some variates because of the random seed. If the average accuracy you got is 91.2%, why i train the model two times without any change, and the results both around 90%.
